packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.styles.ts

```diff
@@ -1,17 +1,75 @@
 import { IDatePickerStyleProps, IDatePickerStyles } from './DatePicker.types';
+import { IStyle, normalize, getGlobalClassNames } from '../../Styling';
 
-export const getStyles = (props: IDatePickerStyleProps): IDatePickerStyles => {
-  const { className } = props;
+const GlobalClassNames = {
+  root: 'ms-DatePicker',
+  textField: 'ms-TextField',
```

Comment:
duplicate - 'ms-TextField' comes from the TextField component root, so when they combine you render two 'ms-TextField' s - the scss implementation didn't have a class for this element just styles.

Reply:
Yeah, this className should be removed